slack_autoarchive.py

```diff
@@ -231,11 +245,17 @@ def main(self):
         whitelist_keywords = self.get_whitelist_keywords()
         alert_templates = self.get_channel_alerts()
         archived_channels = []
+        max_archived_channels = self.settings.get('max_channels_archived')
```

Comment:
Why not `max_channels_archived`? It's a local var anyway, might as well match the name?